extensions/2.0/Khronos/KHR_materials_volume_scatter/README.md

```diff
@@ -0,0 +1,237 @@
+
+$$
+p(\mathbf{v},\mathbf{l}; g) = \frac{1}{4 \pi} \frac{1-g^2}{(1+g^2+2g(\mathbf{v} \cdot \mathbf{l}))^{3/2}}
+$$
+
+For any pair of incident and outgoing directions $\mathbf{v}$ and $\mathbf{l}$.
+
+<table>
+  <tr>
+    <td><img src="figures/sss_anisotropy_-0.95.png"/></td>
+    <td><img src="figures/sss_anisotropy_0.0.png"/></td>
+    <td><img src="figures/sss_anisotropy_0.95.png"/></td>
+  </tr>
+  <tr>
+    <td align="center">g = -0.95</td>
+    <td align="center">g = 0.0</td>
+    <td align="center">g = 0.95</td>
+  </tr>
+  <tr>
+    <td colspan="3" align="center">
+      <em>Top-lit, dense scattering medium for different scatter anisotropy values.</em>
+    </td>
+  </tr>
+</table>
+
+## Scattering & (Diffuse-)Transmission
+
+For dense subsurface scattering materials like skin or wax (where the scattering coefficient $\sigma_s$​ is high), it is recommended to use `KHR_materials_diffuse_transmission` over `KHR_materials_transmission`. In such cases, the visual difference between diffuse and regular transmission is minimal, as volume scattering primarily determines the appearance, with boundary interactions having a small effect.
+
+Using diffuse transmission signals the renderer that the material is dense, eliminating the need to analyze geometry and scattering distance. For instance, a small object with low scattering might appear transparent, but a larger one would appear denser at the same scattering coefficient. Diffuse transmission also enables a translucent look regardless of object size.
+
+This cue allows renderers to use diffusion approximation rather than random walk subsurface scattering, producing near-ground-truth results for dense materials with greater efficiency, essential for both real-time and offline rendering. [Christensen and Burley (2015)](#ChristensenBurley2015) demonstrate the mapping of physical parameters to diffusion-based reflectance profiles, while [Jimenez et al. (2015)](#Jimenez2015) introduce a real-time method using a separable kernel for reflectance profile rendering.
+
+<figure style="text-align:center">
+<img src="./figures/transmission-translucency.png"/>
+<figcaption><em>Comparison of combining denes volume scattering with either transmission or diffuse transmission. Left: Rough transmission and scattering. Middle: Diffuse transmission and scattering. Right: Diffuse transmission without scattering using a thin-walled material. Colors have been adjusted manually in order to account for differences in distances and to minimize the impact of energy loss from the rough microfacet BTDF.</em></figcaption>
+</figure>
+
+## Schema
+
+- [glTF.KHR_materials_subsurface.schema.json](schema/glTF.KHR_materials_subsurface.schema.json)
```

Comment:
The schema filename here should begin with `material.*` instead of `glTF.*` now. This is a relatively recent change applied to other material schemas in this repo.